SmartDeviceLinkTests/SwiftSpec.swift

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
This spec file does not contain any test, maybe it should be removed?

Reply:
Nope, it's suggested to exist as I mentioned in the PR change notes. https://github.com/Quick/Quick/blob/main/Documentation/en-us/QuickInObjectiveC.md#your-test-target-must-include-at-least-one-swift-file